### ISSUE TITLE
contrib-media: Keyboard navigation not possible with video items on Safari 10.0.2

### ISSUE THREAD

**Issue by joewickert** [adaptlearning/adapt_framework#1406]
Think I've just hit a bug with Safari Version 10.0.2 (11602.3.12.0.1) on El Capitan.

With accessibility turned on, the media player breaks tabbing navigation. The media window is highlighted but the play/cc buttons can't be activated and pressing tab again takes the user out of the page into the Safari bookmark bar.

Means that any components after the Media player can't be accessed with keyboard.

I hit the issue in the Authoring Tool. But just replicated with a fresh install of the demo course using CLI.

I can't play or navigate past the video with keyboard only.

**Comment by tomgreenfield** [comment]
It looks like this issue has been fixed in MediaElement v3. Upgrading will involve dropping support for IE8.

@moloko, are we ready to take the plunge?

**Comment by brian-learningpool** [comment]
I don't think we can just drop support for IE 8 mid-version like that.  

If MediaElement v3 is a drop in replacement, is it possible to have two versions co-exist for IE 8 support?

**Comment by moloko** [comment]
Can someone check to see if this is still an issue with Adapt v4?

**Comment by joewickert** [comment]
I've spent a couple of hours on it this morning. I can't replicate the bug as it seems to be impossible to get Safari version 10 to test. I have an old machine running El Capitan that had Safari 9.

I tested on that and the tabbing worked perfectly with Adapt 2 demo course. (tabbing has to activated in safari advanced preferences).

I ran system updates which brought Safari up to version 11 (no option to install 10) and tabbing  works fine in Adapt 2 demo course. 

I created an Adapt 4 demo course from the acessibility4 branch and  uploaded to web server and the tabbing worked fine with the media element.

I did have issues with Safari 9 and 11 accesing courses from local python server. WSOD. But that's something else I guess.

So it's no longer an issue as far as I can tell. Must have been something specific to safari 10.

I did notice that tabbing to the presentation page went straight to Narrative item 1, probably as content above didn't have aria labels.

Thanks, i'll close the issue.
